smoke-tests-otel-starter/spring-smoke-testing/src/main/java/io/opentelemetry/spring/smoketest/AbstractSpringStarterSmokeTest.java

```diff
@@ -22,9 +23,17 @@ public abstract class AbstractSpringStarterSmokeTest {
 
   protected SpringSmokeTestRunner testing;
 
+  @BeforeAll
+  static void beforeAll() {
+    SpringSmokeTestRunner.resetExporters();
+  }
+
   @BeforeEach
   void setUpTesting() {
-    testing = new SpringSmokeTestRunner(openTelemetry);
+    if (openTelemetry != null) {
```

Comment:
Could you please comment this `if` statement?

If `OpenTelemetry` does not exist, the instrumentation can't work. Perhaps the test should fail in this case.